README.md

```diff
@@ -47,32 +54,30 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: GitHub Sync to Upstream Repository
-        uses: dabreadman/sync-upstream-repo@v1.3.0
+        uses: z103db/sync-upstream-repo@v2.0.0
         with: 
-          upstream_repo: ${{ env.UPSTREAM_URL }}
+          upstream_repo_url: ${{ env.UPSTREAM_REPO_URL }}
           upstream_branch: ${{ env.UPSTREAM_BRANCH }}
+          downstream_repo_url: ${{ env.DOwNSTREAM_REPO_URL }}
           downstream_branch: ${{ env.DOWNSTREAM_BRANCH }}
-          token: ${{ env.WORKFLOW_TOKEN }}
+          token: ${{ secret.GHA_TOKEN }}
```

Comment:
Maybe specify that this defaults to `github.token` (if adding it as default value for token in `action.yaml)